%YAML:1.0

# camera intrinsics (for left) mm
camera.fx: 351.24
camera.fy: 351.96
camera.cx: 315.28
camera.cy: 243.18

# show image
SHOW_TRACKER: 1


# feature tracker parameters
Frequence: 10                       # the fequence of publish feature tracker
BorderSize: 1                       #
FeatureMinDis: 40                   #
MaxFeatureCnt: 100                  #
TrackerSize: 50

F_THRESHOLD: 1.0
EQUALIZE: 1